libr/core/config.c

```diff
@@ -1,12 +1,35 @@
 /* radare - LGPL - Copyright 2009-2016 - pancake */
-
 #include <r_core.h>
 
 #define SETI(x,y,z) r_config_node_desc(r_config_set_i(cfg,x,y), z);
 #define SETICB(w,x,y,z) r_config_node_desc(r_config_set_i_cb(cfg,w,x,y), z);
 #define SETPREF(x,y,z) r_config_node_desc(r_config_set(cfg,x,y), z);
 #define SETCB(w,x,y,z) r_config_node_desc(r_config_set_cb(cfg,w,x,y), z);
 
+/*------------------------------------------------------------------------------------------*/
+
+R_API void r_config_set_sort_column(char *column){
+	if (!column) {
+		r_anal_compare = &compareName;
+	}
+        if (!strcmpi (column,"name")) {
```

Comment:
i would certainly prefer to have a for loop iterating over an array of structs with columname and sortcallback